### DISCUSSION TITLE
We REALLY need binary EXE file 🔑 .

### DISCUSSION THREAD

**Discussion by KoghaandSooga** [86Box/86Box#3595] [Ideas]
Or this project will forever be held under lock and 🔑 as not all of us have the ability to compile files nor have time to do something for an unknown program.  So again do you think there is a rough date for an EXE anytime within the next year/year and a half?  I looked under actions tab and all I found were merges for 86box whatever that meant.   It seems it should be a pretty much a basic thing regardless how complete/incomplete PCBOX actually is.   Too bad I can never try it out.  

Someone else made a thread wishing to spam binary builds so other devs can get on board (Though I doubt any exist) Devs that is knowing our generation.

**Comment by lemondrops** [comment]
You probably meant to post this there: https://github.com/PCBox/PCBox/discussions.

Unless you really meant \_**86**\_Box and not \_**PC**\_Box, in which case, stable release binaries are available [here](https://github.com/86Box/86Box/releases) and latest development builds can be downloaded [here](https://ci.86box.net/job/86Box/).

**Comment by KoghaandSooga** [comment]
Sorry. This is pretty confusing compared to standard forums.

**Comment by KoghaandSooga** [comment]
I just discovered the close function.
